PocketKit/Sources/PocketKit/Main/RegularHomeCoordinator.swift

```diff
@@ -118,12 +114,37 @@ extension RegularHomeCoordinator {
 extension RegularHomeCoordinator {
     private func show(_ readable: ReadableType?) {
         switch readable {
-        case .recommendation(let recommendation):
-            show(recommendation)
-        case .savedItem(let savedItem):
-            show(savedItem)
+        case .savedItem(let viewModel):
+            show(viewModel)
+        case .recommendation(let viewModel):
+            show(viewModel)
+        case .webViewRecommendation(let viewModel):
+            viewModel.$presentedAlert.sink { [weak self] alert in
+                self?.present(alert)
+            }.store(in: &readerSubscriptions)
+
+            viewModel.$selectedRecommendationToReport.sink { [weak self] recommendation in
+                self?.report(recommendation)
+            }.store(in: &slateDetailSubscriptions)
+
+            push(viewModel.url)
```

Comment:
Similar to above, can we extract this into its own function (for readability)?

Reply:
@dskuza done.